sridha2412/assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# print ancestors of given key in a binary tree
```

Comment:
I suggest you do unit test :) At the last mentors meeting, it was decided that unit tests will only be required for the next assignment, but I suggest you still do it for this assignment.